### ISSUE TITLE
Clean up our Status tags for issues

### ISSUE THREAD

**Issue by kvark** [gfx-rs/gfx#934]
I've been assigning `Status: Ready` to most of the issues, now I realize this is misleading. We need to rethink what the status is and update the issues.

**Comment by Bastacyclop** [comment]
@kvark Here is what I think about the different statuses:
- `broken`: I don't know what this is for.
- `blocked`: progress on this issue is blocked by another.
- `duplicate`: pretty obvious.
- `invalid`: I don't know what this is for.
- `ready`: The issue is ready to be fixed.
- `working`: The issue is being fixed.

Some possible names:
- `ready` -> `fixable`/`workable`/`work-ready`
- `working` -> `fixing`/`working`/`[work ]in progress`

**Comment by Bastacyclop** [comment]
`ready` is now `work-ready`.
`working` is now `in progress`.
I still don't know what is the meaning of `broken` and `invalid` though.

**Comment by kvark** [comment]
Broken should be killed.
Invalid is for stuff outside of our domain, e.g. an OpenGL rendering issue, or the meat being undercooked  for dinner. Also known as Won't Fix.

**Comment by Bastacyclop** [comment]
`broken` is removed.
What do you think about `invalid` -> `out of scope` ?

**Comment by kvark** [comment]
Sure

**Comment by Bastacyclop** [comment]
that's done :)
